### DISCUSSION TITLE
Possibile sync problem - popup window

### DISCUSSION THREAD

**Discussion by adWroc** [NightscoutFoundation/xDrip#3583] [Q&A]
Today this message started appearing.  Anyone know what the problem is? Every time I click YES, but after some time it appears again.
![IMG-20240723-WA0003](https://github.com/user-attachments/assets/f60d271e-d6da-41f0-b871-8ea26c282741)

**Comment by Navid200** [comment]
You need to update to the xDrip release of July 23, 2024 or any after that.

**Reply by adWroc** [reply]
Thank You. I updated the app on my phone, but now I have a problem updating it on my watch. The update process goes into a loop: sending the last version, updating %, and again and again. I tried using wear installer 2 but after selecting the xdrip app I get an error: the app is not intended for the watch.
